psrelease.psm1

```diff
@@ -1,81 +1,13 @@
-# VSTS task states: Succeeded|SucceededWithIssues|Failed|Cancelled|Skipped
-$succeededStateName = 'Succeeded'
-$warningStateName = 'SucceededWithIssues'
-$errorStateName = 'Failed'
-
-# store the current state used by *-VstsTaskState and Write-VstsMessage
-$script:taskstate = $succeededStateName
+Import-Module "$PSScriptRoot/vstsBuild" -Scope Global
+Import-Module "$PSScriptRoot/dockerBasedBuild"
+Import-Module "$PSScriptRoot/dockerBasedBuild/dockerBasedBuild.common.psm1"
```

Comment:
It seems `dockerBasedBuild.common.psm1` needs to be imported by all other modules (psrelease.psm1, dockerBaseBuild.psm1 and vstsBuild.psm1). Why do you want it to be in `dockerBasedBuild` module folder instead of a totally separate module?

Reply:
PSRelease should be short-lived (only there for compat until we move to dockerBasedBuild) and I have removed vstsBuild's dependency on this module.